### PR TITLE
ss-2: common styles

### PR DESCRIPTION
**Pull request by Evcat6**

### CODE REVIEW

File: frontend/src/assets/css/margin-variables.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+:root {
+    --m-025: 0.25rem;
+    --m-05: 0.5rem;
+    --m-1: 1rem;
+    --m-15: 1.5rem;
+    --m-2: 2rem;
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/margin-variables.css:6]:
```suggestion
    --m-1: 0.25rem;
    --m-2: 0.5rem;
    --m-3: 1rem;
    --m-4: 1.5rem;
    --m-5: 2rem;
```

File: frontend/src/assets/css/padding-variables.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+:root {
+    --p-025: 0.25rem;
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/padding-variables.css:2]: The same as with margins

File: frontend/src/assets/css/typography-variables.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
+:root {
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/typography-variables.css:1]: use 16px as the base size. And all other sizes try to extend from it
the same for line-height

File: frontend/src/assets/css/typography-variables.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    --font-weight-400: 400;
+    --font-weight-500: 500;
+    --font-family: 'Rubik', sans-serif;
+
+    /* Heading 1 */
+    --font-size-level-1-heading: 44px;
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/typography-variables.css:7]: base size 16px
```suggestion
    --font-size-level-1-heading: 2.75rem;
```

File: frontend/src/assets/css/typography-variables.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    --line-height-quotation: 24px;
+
+    /* Input label */
+    --font-size-input-label: 16px;
+    --font-weight-input-label: 400;
+    --line-height-input-label: 16px;
+
+    /* Input text */
+    --font-size-input: 15px;
+    --letter-spacing-input: -1%;
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/typography-variables.css:60]: I think it unnecessary

File: frontend/src/assets/css/color-variables.css [hunk 6]
```diff
@@ -0,0 +1,29 @@
+:root {
+    /* Background */
+    --background-blue-600: #163676;
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/color-variables.css:3]:
```suggestion
    --color-blue-600: #163676;
```

File: frontend/src/assets/css/color-variables.css [hunk 7]
```diff
@@ -0,0 +1,29 @@
+:root {
+    /* Background */
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/color-variables.css:2]: make just colors. We don't need to specify it for fonts and background

File: frontend/src/assets/css/color-variables.css [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    --background-blue-100: #EFF3FF;
+
+    /* Font */
+    --text-color-gray-400: #130138;
+    --text-color-gray-300: #5A6474;
+    --text-color-gray-200: #9AACBD;
+    --text-color-white-100: #FFFFFF;
+
+    /* Brand */
+    --brand-color-turquoise: linear-gradient(95.77deg, #00D7BD -14.06%, #03BFD9 101.51%);
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/color-variables.css:15]:
```suggestion
    --gradient-turquoise: linear-gradient(95.77deg, #00D7BD -14.06%, #03BFD9 101.51%);
```

File: frontend/src/assets/css/breakpoint-variables.css [hunk 9]
```diff
@@ -0,0 +1,7 @@
+:root {
+    --breakpoint-sm: 576px;
+    --breakpoint-md: 768px;
+    --breakpoint-lg: 992px;
+    --breakpoint-xl: 1200px;
+    --breakpoint-xxl: 1400px;
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/breakpoint-variables.css:6]:
```suggestion
    --sm: 576px;
    --md: 768px;
    --lg: 992px;
    --xl: 1200px;
```

4 breakpoint enough

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/assets/css/margin-variables.css (hunk 1 above)
Comment (Evcat6) [reply frontend/src/assets/css/margin-variables.css]: Done in commit [d7dfd60](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/d7dfd60823e771fb8617e2e4c36d6cf2f6dff46a)

File: frontend/src/assets/css/padding-variables.css (hunk 2 above)
Comment (Evcat6) [reply frontend/src/assets/css/padding-variables.css]: Done in commit [b72b2bf](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/b72b2bf44b80194ef93f40a62fa9cc92f9bb6024)

File: frontend/src/assets/css/breakpoint-variables.css (hunk 9 above)
Comment (Evcat6) [reply frontend/src/assets/css/breakpoint-variables.css]: Done in commit [79c8d92](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/79c8d924c82ac336a7e505cad7d9b7de4a8f2db8)

File: frontend/src/assets/css/color-variables.css (hunk 8 above)
Comment (Evcat6) [reply frontend/src/assets/css/color-variables.css]: Done in commit [03db74d](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/03db74d7ec35888e47792a6337700fc5e0d1a4a5)

File: frontend/src/assets/css/color-variables.css (hunk 7 above)
Comment (Evcat6) [reply frontend/src/assets/css/color-variables.css]: Done in commit [03db74d](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/03db74d7ec35888e47792a6337700fc5e0d1a4a5)

File: frontend/src/assets/css/color-variables.css (hunk 6 above)
Comment (Evcat6) [reply frontend/src/assets/css/color-variables.css]: Done in commit [03db74d](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/03db74d7ec35888e47792a6337700fc5e0d1a4a5)

File: frontend/src/assets/css/typography-variables.css (hunk 4 above)
Comment (Evcat6) [reply frontend/src/assets/css/typography-variables.css]: Done in commit [c829865](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/c82986598eeb62c53f77120da7f753c700286dd9)

File: frontend/src/assets/css/typography-variables.css (hunk 5 above)
Comment (Evcat6) [reply frontend/src/assets/css/typography-variables.css]: Done in commit [c829865](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/c82986598eeb62c53f77120da7f753c700286dd9)

File: frontend/src/assets/css/typography-variables.css (hunk 3 above)
Comment (Evcat6) [reply frontend/src/assets/css/typography-variables.css:1]: Done in commit [1453a79](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/1453a7989a7243d01e5231c26d0ce80bef572bfa)

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
In general looks good

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**